src/components/TimelineVertical.tsx

```diff
@@ -0,0 +1,135 @@
+import { VerticalTimeline, VerticalTimelineElement } from 'react-vertical-timeline-component';
+import 'react-vertical-timeline-component/style.min.css';
+import React from 'react';
+import { ITimeline } from '../types/types';
+
+interface Props {
+    data: ITimeline[];
+}
+
+const Timeline: React.FC<Props> = ({ data }) => {
+    return (
+        <>
+            <VerticalTimeline>
+                {data.map(item => (
+
+                    <VerticalTimelineElement
+                        className="vertical-timeline-element--work"
+                        //contentStyle={{ background: 'rgb(33, 150, 243)', color: '#fff' }}
+                        contentArrowStyle={{ borderRight: '7px solid  rgb(33, 150, 243)' }}
```

Comment:
magical numbers is it really ok?

Reply:
I delegated the values to a property for now, but later I will change all the magic numbers onto a theme page